src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -102,11 +102,11 @@ def __init__(self,
     self.address = address  # Set in case a TargetDefinitionException is thrown early
     payload = payload or Payload()
     excludes = ExcludesField(self.assert_list(excludes, expected_type=Exclude, key_arg='excludes'))
-
     payload.add_fields({
       'sources': self.create_sources_field(sources, address.spec_path, key_arg='sources'),
       'provides': provides,
       'excludes': excludes,
+      'resources': PrimitiveField(self.assert_list(resources, key_arg='resources')),
```

Comment:
...I guess it's just the address spec strings. Those should be stable. Alright, no big.